lib/pygments/popen.rb

```diff
@@ -215,7 +215,7 @@ def mentos(method, args=[], kwargs={}, original_code=nil)
 
       begin
         # Timeout requests that take too long.
-        timeout_time = ENV["MENTOS_TIMEOUT"] || 8
+	timeout_time = Integer(ENV["MENTOS_TIMEOUT"]) rescue 8
```

Comment:
Oops, I fixed the whitespace.

Also, I added a comment to clarify that I think the correct behavior for an invalid `MENTOS_TIMEOUT` should be the default, `8`, rather than `0`, as would be the case if just using `to_i`.